package/MDAnalysis/core/groups.py

```diff
@@ -736,15 +748,13 @@ def center(self, weights, pbc=None, compound='group', unwrap=False):
 
         comp = compound.lower()
         if comp == 'group':
-            if unwrap and pbc:
-                raise ValueError("'unwrap' and 'pbc' cannot be true at the same time "
-                                 "for compound='group")
-            if unwrap:
-                coords = atoms.unwrap(compound=comp, reference=None, inplace=False)
             if pbc:
                 coords = atoms.pack_into_box(inplace=False)
             else:
                 coords = atoms.positions
+            if unwrap:
```

Comment:
This logic isn't quite right.  For the case where `pbc=False, unwrap=True` then `coords` is generated twice.  Use an `elif` to fix this